code/modules/mob/living/simple_animal/hostile/headcrab.dm

```diff
@@ -69,7 +69,9 @@
 		qdel(src)
 
 /obj/item/organ/internal/body_egg/changeling_egg/proc/Pop()
-	var/mob/living/carbon/human/species/monkey/spawned_monkey = new(owner)
+	var/mob/living/carbon/human/spawned_monkey = new(owner)
+	spawned_monkey.set_species(/datum/species/monkey)
+	spawned_monkey.regenerate_icons()
```

Comment:
🤔 set species should handle all icon updates. regenerate icons says explicitly to not use it.

Reply:
Ahh I see. Thank you for pointing it out!